Puppetfile

```diff
@@ -12,7 +12,7 @@ mod 'puppetlabs/concat'
 mod 'puppetlabs/gcc'
 mod 'puppetlabs/java_ks'
 mod 'puppetlabs/mysql'
-mod 'puppetlabs/postgresql', '4.1.0'
+mod 'puppetlabs/postgresql', '4.8.0'
```

Comment:
should we remove the old versions tarball in this PR if we're adding the new one?